charts/osdfir-infrastructure/README.md

```diff
@@ -349,9 +397,12 @@ Please be cautious before doing it.
 | ---------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- | ------- |
 | `global.timesketch.enabled`              | Enables the Timesketch deployment (only used in the main OSDFIR Infrastructure Helm chart)                                         | `true`  |
 | `global.yeti.enabled`                    | Enables the Yeti deployment (only used in the main OSDFIR Infrastructure Helm chart)                                               | `true`  |
+| `global.openrelik.enabled`               | Enables the OpenRelik deployment (only used in the main OSDFIR Infrastructure Helm chart)                                          | `true`  |
 | `global.ingress.enabled`                 | Enable the global loadbalancer for external access (only used in the main OSDFIR Infrastructure Helm chart)                        | `false` |
 | `global.ingress.timesketchHost`          | Domain name Timesketch will be hosted under                                                                                        | `""`    |
 | `global.ingress.yetiHost`                | Domain name Yeti will be hosted under                                                                                              | `""`    |
+| `global.ingress.openRelikFrontendHost`   | Domain name the OpenRelik UI will be hosted under                                                                                  | `""`    |
```

Comment:
I see it defaults to localhost if not set, might be usefull to mention the default value here?

Reply:
Yes agreed, I updated to mention (defaults to localhost)